pkg/features/notifications.go

```diff
@@ -52,7 +52,9 @@ func RunNotifications(organization string, team string, author string, teamMembe
 			createdAt,
 		)
 		if err != nil {
-			panic(err)
+			slog.Debug("Failed to fetch Pull Requests because of an error. Retrying in 5 mins...", "Error", err)
+			time.Sleep(300 * time.Second)
```

Comment:
I feel like often this happens when my laptop awakes from suspend and if I restart the application it works first try. So I think 5 minutes is too long. I would rather try 5 seconds and maybe adjust later (or introduce some back off logic)